### PR TITLE
Post Kubecon project update

### PR DESCRIPTION
**Pull request by mtwo**
To be published at 10:00 AM ET

### CODE REVIEW

File: content/en/blog/2022/kubecon-na-project-update.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+author: Morgan McLean
+---
+
+2022 has been an incredible year for OpenTelemetry. Metrics became a first-class signal type, and are being used on production services and infrastructure alongside OpenTelemetry’s existing distributed tracing support to send critical performance data to any observability backend for processing. Everywhere we look we see OpenTelemetry being tested, rolled out, or already in use in organizations everywhere, from the largest to smallest, from the most cutting edge to the historically cautious.
+
+Since January, we’ve delivered:
+
+- Metrics: defined in the specification, and delivered in the Java, JS, .Net, and Python SDKs and instrumentation, with support all the way through the Collector and protocol. More language implementations are on the way.
+- More instrumentation for all languages.
+- [A great demo application](/announcing-opentelemetry-demo-release), which includes services written in every supported language, already instrumented with OpenTelemetry. This is a great way to see practical examples of OpenTelemetry that you can learn from or experiment with, and the demo can also be used to test out different observability analytics systems.
```
Comment (mtwo) [inline content/en/blog/2022/kubecon-na-project-update.md:14]:
```suggestion
- [A great demo application](../announcing-opentelemetry-demo-release), which includes services written in every supported language, already instrumented with OpenTelemetry. This is a great way to see practical examples of OpenTelemetry that you can learn from or experiment with, and the demo can also be used to test out different observability analytics systems.
```

**Review by svrnm — APPROVED** [review]
looks good to me

**Review by austinlparker — APPROVED** [review]

File: content/en/blog/2022/kubecon-na-project-update.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+linkTitle: Project Update
+date: 2022-10-26
+author: Morgan McLean
+---
+
+2022 has been an incredible year for OpenTelemetry. Metrics became a first-class signal type, and are being used on production services and infrastructure alongside OpenTelemetry’s existing distributed tracing support to send critical performance data to any observability backend for processing. Everywhere we look we see OpenTelemetry being tested, rolled out, or already in use in organizations everywhere, from the largest to smallest, from the most cutting edge to the historically cautious.
+
+Since January, we’ve delivered:
+
+- Metrics: defined in the specification, and delivered in the Java, JS, .Net, and Python SDKs and instrumentation, with support all the way through the Collector and protocol. More language implementations are on the way.
```
Comment (cartermp) [inline content/en/blog/2022/kubecon-na-project-update.md:12]:
```suggestion
- Metrics: defined in the specification, and delivered in the Java, JS, .NET, and Python SDKs and instrumentation, with support all the way through the Collector and protocol. More language implementations are on the way.
```

File: content/en/blog/2022/kubecon-na-project-update.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+- More instrumentation for all languages.
+- [A great demo application](../announcing-opentelemetry-demo-release), which includes services written in every supported language, already instrumented with OpenTelemetry. This is a great way to see practical examples of OpenTelemetry that you can learn from or experiment with, and the demo can also be used to test out different observability analytics systems.
+- Tracing stability in C++, Erlang, and a number of other new languages.
+- Progress on Ruby, Erlang, Swift, .Net auto instrumentation.
```
Comment (cartermp) [inline content/en/blog/2022/kubecon-na-project-update.md:16]:
```suggestion
- Progress on Ruby, Erlang, Swift, .NET auto instrumentation.
```

**Review by cartermp — APPROVED** [review]
Looks good overall, just gotta spell .NET with all caps :)

(some of the community feels it should be called `dotnet` like the CLI, but alas, it still suffers from a failed early-2000s marketing initiative)

**Pull request merged**